pkg/api/latest/latest.go

```diff
@@ -122,9 +123,15 @@ func init() {
 		"Namespace": true,
 	}
 
+	// these kidns should be excluded from the list of resources
+	ignoredKinds := util.NewStringSet("ListOptions", "DeleteOptions", "Status", "ContainerManifest")
```

Comment:
These are not "resources", just "kinds".  Objects that can be REST resources are pods, services, nodes.  Objects that can only be returned from REST resources, or are in the scheme but never passed over the wire, are not "resources" (as in, they are exposed to a client via the API).